latest/Dockerfile.rhel7

```diff
@@ -13,7 +13,7 @@ LABEL summary="$SUMMARY" \
       io.openshift.expose-services="27017:mongodb" \
       io.openshift.tags="database,mongodb,rh-mongodb34" \
       com.redhat.component="rh-mongodb34-docker" \
-      name="rhscl/mongodb-34-rhel7" \
+      name="rhscl-beta/mongodb-34-rhel7" \
```

Comment:
We are using the regular `rhscl/` namespace this time around so no change needed here.

Reply:
Thanks, fixed.